rise-rss.html

```diff
@@ -357,13 +357,6 @@
        *
        */
       go: function() {
-        if (!offlineWindow) {
```

Comment:
@stulees Can you review this please? I believe that we got rid of offline player logic in the Storage component, and so I'm wondering if it's safe to rip out this entire check here. Thx.